camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/mapred/EtlInputFormat.java

```diff
@@ -135,16 +135,21 @@ public static void setLogger(Logger log) {
     return topicMetadataList;
   }
 
-  private SimpleConsumer createConsumer(JobContext context, String broker) {
+  private SimpleConsumer createBrokerConsumer(JobContext context, String broker) {
```

Comment:
I do have a createSimpleConsumer() method later in the file.

The createBrokerConsumer is just a simple wrapper to call createSimpleConsumer with broker information.

The createSimpleConsumer() was first created to be an instance method so the subclass can override, the first iteration of the framework was using subclass: TestEtlInputFormat, we might go back to that route.  For this reason, I think it's better to keep it as instance method.